docker/k8s/tcs_config.toml

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[KvStorage]
+StoragePath = "config/Kv_Shared_tmp"
+# the remote version is of higher priority if enabled
+remote_url = "http://lmdb:9090"
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# --------------------------------------------------------------
+# EnclaveManager -- configuration of SGX Enclave EnclaveManager
+# --------------------------------------------------------------
+[EnclaveManager]
+sleep_interval = "10"
+
+[Listener]
+listener_port = 1947
+max_work_order_count = 10
+
+# --------------------------------------------------
+# EnclaveModule -- configuration of the SGX Enclave
+# --------------------------------------------------
+[EnclaveModule]
+# spid is a 32-digit hex string tied to the enclave implementation
+# Replace dummy SPID with value obtained after subscription to run TCS 
+# in SGX HW mode
+spid = "DEADBEEF00000000DEADBEEF00000000"
+
+num_of_enclaves = "1"
+
+# ias_url is the URL of the Intel Attestation Service (IAS) server.
+ias_url = "https://api.trustedservices.intel.com/sgx/dev"
+
+# Proxy for https. Change to your corporate proxy or comment out below line for
+# direct Internet connections.
+https_proxy = "http://proxy-iind.intel.com:912/"
```

Comment:
For Docker, the proxy can be set with `~/.docker/config.json`
So please comment this line out.

The file format is documented at in the section "Configure the Docker client"
for Docker 17.07 and higher in "Configure Docker to use a proxy server" at
https://docs.docker.com/network/proxy/#configure-the-docker-client

```
{
 "proxies":
 {
   "default":
   {
     "httpsProxy": "http://proxy-iind.intel.com:912"
   }
 }
}
```

Reply:
done https://github.com/hyperledger-labs/trusted-compute-framework/blob/cfb80a212a0dedf4a28d124380afeefd5d6335b8/docker/k8s/tcs_config.toml#L51-L53